src/test/unit/process-profile.test.js

```diff
@@ -183,14 +183,12 @@ describe('gecko counters processing', function() {
     const childCounter = createGeckoCounter(findMainThread(childGeckoProfile));
 
     // Due to a bug in Gecko, it's possible that we have a counter that has no
-    // sample data. This is likely a Gecko problem that we'll need to fix.
-    // See https://github.com/firefox-devtools/profiler/issues/2248 and
-    // https://bugzilla.mozilla.org/show_bug.cgi?id=1584190
+    // sample data. In that case, we'll have an empty sample_groups array.
```

Comment:
Same remark as above: I don't think this is a bug, it just happens. (unless you tell me otherwise)